pkg/service/replay/replay.go

```diff
@@ -77,20 +77,37 @@ func (r *replayer) Start(ctx context.Context) error {
 		}
 	}()
 
-	// BootReplay will start the hooks and proxy and return the testRunID and appID
-	testRunID, appID, hookCancel, err := r.BootReplay(ctx)
+	absPath, err := filepath.Abs(r.config.Path)
```

Comment:
please do these checks in [validation of flags](https://github.com/keploy/keploy/blob/48ec8ba3ba8b5ea1389768ea0b3a559404d76e6e/cli/provider/cmd.go#L286)